### PR TITLE
Provide some labels for helping in development

### PR DESCRIPTION
**Pull request by danidoni**

### CODE REVIEW

**Comment by hennevogel** [comment]
I don't get those rake tasks that wrap `create`. What are they good for?

**Comment by danidoni** [comment]
> I don't get those rake tasks that wrap `create`. What are they good for?

So you can do `rake dev:label_templates:data[10]` and have 10 label templates ready on dev

**Comment by danidoni** [comment]
> I don't get those rake tasks that wrap `create`. What are they good for?

In the end I remove the task. It's not really providing much.

File: src/api/spec/factories/label_template.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :label_template do
+    name { Faker::Lorem.word.capitalize }
+    color { (rand * 0xffffff).to_i.to_s(16) } # Random value from 0x000000 - 0xffffff
```
[the remark this answers is not in the document]
Comment (danidoni) [reply src/api/spec/factories/label_template.rb:4]: Yes it does indeed.

**Comment by saraycp** [comment]
I re-ran the feature specs. Don't forget to squash the commits, please.

File: src/api/lib/tasks/dev.rake [hunk 2]
```diff
@@ -224,6 +224,8 @@ namespace :dev do
       # Trigger package builds for home:Admin
       home_admin.store
 
+      5.times { create(:label_template, project: home_admin) }
```
Comment (hennevogel) [inline src/api/lib/tasks/dev.rake:227]: https://thoughtbot.github.io/factory_bot/building-or-creating-multiple-records/summary.html

**Review by hennevogel — CHANGES_REQUESTED** [review]

File: src/api/lib/tasks/dev.rake (hunk 2 above)
Comment (danidoni) [reply src/api/lib/tasks/dev.rake]: Applied

**Review by saraycp — APPROVED** [review]

**Review by hennevogel — APPROVED** [review]

**Pull request merged**